src/frontend/citizen-portal/src/app/api/generate.sh

```diff
@@ -0,0 +1,3 @@
+docker pull openapitools/openapi-generator-cli:latest
+
+docker run --rm -v "$(pwd)":/local openapitools/openapi-generator-cli generate -i /local/client-api.json -g typescript-angular -t /local/templates --additional-properties=modelFileSuffix=.model -o /local
```

Comment:
It may be cleaner to have a `yarn api:generate` target rather than a bat/sh file the uses docker to download and run openapi

Reply:
Done in latest commit